### PR TITLE
[T4A6][F11-B3]Tiong YaoCong

### PR DESCRIPTION
**Pull request by ChaseTiong**
## Create of new Junit Test
- UtilsTest created
- Successfully test all Utils class methods

### CODE REVIEW

**Comment by chasexyaocong** [comment]
Ready For Review

**Comment by ChaseTiong** [comment]
Review

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:8]: add comments.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+public class UtilsTest {
+
+	@Test
+	public void testIsAnyNull() {
+		Object[] testingObjects = new Object[3];
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:13]: naming not to standard

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+		testingObjects[0] = null;
+		assertTrue(Utils.isAnyNull(testingObjects[0]));
+		testingObjects[1] = "abc";
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:16]: refactor test cases. each case should be a separate method.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:4]: avoid \* imports. use eclipse shortcut to import easily.

**Comment by cmkumar87** [comment]
@ChaseYaoCong Pls close PR. Thanks!

**Pull request closed without merging**